app/Console/Commands/ImportNasspGroupsCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Exception;
+use League\Csv\Reader;
+use Rogue\Models\Group;
+use Rogue\Models\GroupType;
+use Illuminate\Console\Command;
+
+class ImportNasspGroupsCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:nassp-groups-import {path}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create National Association of Secondary School Principals group types and groups from a CSV.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Returns a trimmed CSV value, or null if equal to the string "NULL".
+     *
+     * @param string
+     * @return string
+     */
+    public function sanitize($value)
+    {
+        $result = trim($value);
+
+        return $result == 'NULL' ? null : $value;
```

Comment:
Should we be returning the `$result` instead of the `$value` here?

Reply:
Argh yes -- 🦅 👀 thank you